test/integration/fixtures/01-users.js

```diff
@@ -8,19 +8,22 @@ module.exports = ({ Assignments, Users, Projects, bcrypt }) => {
   const users = [
     new User({ email: 'alice@getodk.org', password: 'alice' }, { actor: new Actor({ type: 'user', displayName: 'Alice' }) }),
     new User({ email: 'bob@getodk.org', password: 'bob' }, { actor: new Actor({ type: 'user', displayName: 'Bob' }) }),
-    new User({ email: 'chelsea@getodk.org', password: 'chelsea' }, { actor: new Actor({ type: 'user', displayName: 'Chelsea' }) })
+    new User({ email: 'chelsea@getodk.org', password: 'chelsea' }, { actor: new Actor({ type: 'user', displayName: 'Chelsea' }) }),
+    new User({ email: 'dave@getodk.org', password: 'dave' }, { actor: new Actor({ type: 'user', displayName: 'Dave' }) }),
```

Comment:
After writing the test above, I'm having second thoughts about adding Dave, since Dave is only needed for a couple of specific tests. What do you think about having Alice assign Chelsea the `formfill` and `viewer` roles on project 1, similar to how the test above is structured?

Reply:
i should have put more thought into the fixtures system. as it is, too many tests depend on the exact fixture values today, and it's impractical to ever edit them. probably some system where a test can elect to use a known alternate fixture would be better. there are a lot of eg form/submissions tests that every single one of them add the same test data over and over.

Reply:
I've dispatched with Dave.